File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1158.c
#include <stdio.h>

int main(){
    int n, i, j, x, y, soma;
    scanf("%d", &n);
    for(i=0;i<n;i++){
        scanf("%d %d", &x, &y);
        if(x%2==0){
            x++;
        }
        soma=0;
        for(j=0;j<y;j++){
            soma+=x;
            x+=2;
        }
        printf("%d\n", soma);
    }

    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1158.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
        Scanner ler = new Scanner(System.in);
        
        int n = ler.nextInt(), i, j, x, y, soma;
        for(i=0;i<n;i++){
            x = ler.nextInt();
            y = ler.nextInt();
            if(x%2==0){
                x++;
            }
            soma=0;
            for(j=0;j<y;j++){
                soma+=x;
                x+=2;
            }
            System.out.printf("%d\n", soma);
        }
    }
}